make_profiler/report/script_u.js

```diff
@@ -0,0 +1,45 @@
+function filterTarget(){var t,e,a=document.getElementById("myInput").value.toUpperCase().split(" ").join("_"),n=document.getElementById("statusTable").getElementsByTagName("tr");if(a.startsWith('"')&&a.endsWith('"'))for(a=a.substring(1,a.length-1),e=0;e<n.length;e++)(t=n[e].getElementsByTagName("td")[0])&&((t.childNodes[0].textContent||t.innerText).toUpperCase()===a?n[e].style.display="":n[e].style.display="none");else for(e=0;e<n.length;e++)(t=n[e].getElementsByTagName("td")[0])&&(-1<(t.textContent||t.innerText).toUpperCase().indexOf(a)?n[e].style.display="":n[e].style.display="none")}function doRequest(){var t=new XMLHttpRequest;t.open("GET","report.json",!0),t.onreadystatechange=function(){if(4==this.readyState&&200==this.status){var e=JSON.parse(this.responseText),a=e.status,e=e.pipeline,e=`
```

Comment:
why this script is minified?

Reply:
just minified and uglified, reduced readability. if no need ok with script.js

Reply:
wont use